### DISCUSSION TITLE
README.md is displayed in source code in the repo page

### DISCUSSION THREAD

**Discussion by ntexplorer** [community/community#23256] [New to GitHub]
<p>I’ve written a README.md for my project and push it to my <a href="https://github.com/ntexplorer/JavaCW" title="My Repo" rel="nofollow noopener">repo</a> but it is displayed in the markdown source code rather than the “Preview Mode”. It works fine from my local Markdown editor in which it is displayed in the preview mode. Can’t figure out how to solve this issue :frowning:</p>

**Comment by ntexplorer** [comment, marked as answer]
<p>Added a period at the end of the markdown and re-committed it. Now it works fine… God knows why</p>

**Comment by burbigo1** [comment]
<p>I agree, that’s kinda weird, but the most important is that it works now.</p>

**Comment by ntexplorer** [comment]
<p>And for now, I got a message from GitHub to look into this issue so I reset my repo back to the one with the problem. Feel free to take a peek yourself :slight_smile:</p>
